packages/create-next-app/create-app.ts

```diff
@@ -93,6 +94,15 @@ export async function createApp({
   }
 
   const root = path.resolve(appPath)
+
+  if (!(await isWriteable(path.dirname(root)))) {
+    console.error('The operation was rejected by your operating system.')
```

Comment:
```suggestion
    console.error('The application path is not writable, please check folder permissions and try again.')
```
Should we be more descriptive here and suggest what they need to check, maybe something like this?